### PR TITLE
Corrects error in raw json file and add them to en-corrected folder, …

### PR DESCRIPTION
**Pull request by LeDeep**
…which then gets passed to our script that adds the <p> tags
- Add a folder that contains the corrected raw json
- This corrected json is then run through our script, which combines the files and replaces `\n\n` characters with `<p>` tags.
- Since these corrections will impact additional languages, this corrected file can be sent to translation services for updating

### CODE REVIEW

File: corrected-raw-data/en-corrected/my-history.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      "pagePrompt": "Have you had any medical conditions in the last three years that affected your ability to drive?",
+      "explanationPrompt": "Please explain below."
+    },
+    "cardHistoryPage": {
+      "pagePromptLicense": "Have you ever had a driver license or state-issued ID card?",
+      "pagePromptID": "Have you ever had a California driver license or ID card?",
+      "explanation": "The license or ID card must be issued by a U.S. state or another country.",
+      "explanationPrompt": "Please tell us about your most recent driver license or ID card, even if it's expired.",
+      "numberLabel": "Driver license or ID card number",
+      "stateOrCountryLabel": "Driver license or ID card number",
```
Comment (kyliepace) [inline corrected-raw-data/en-corrected/my-history.json]: I think line 13 should say something else. can this be corrected in this PR?

**Comment by baccigalupi** [comment]
@LeDeep Maybe because the story was written in dev, we just aren't getting to something that will work for our process.

These whole new files can't go to translation. Most of the content has already been translated. Both in a cost and time level, the project can't afford to all the content out for translation when there are typos in the keys, or we want to reorganize the data. 

We are looking for missing content to go in new `raw-data` files. And we are looking for programmatic ways to change the json keys and formatting so that it is how we as programmers would like to use it. The programatic changes can be applied to the translations coming in.

Let's forget about this PR. I will doctor the story carefully to explicitly state what should be done about the surveyed changes in the app `en.json`

**Comment by LeDeep** [comment]
@baccigalupi There seem to be three issues with the translations file.
1. Some of the JSON keys need to be updated so that we can make proper use of them in the application. Like you said, this needs to be done programmatically, perhaps something similar to this PR that was reverted (https://github.com/stateofca/dlite-translations/pull/2).

2. There are some missing translations. It seems like we want to create a new file in the `raw-data` folder that will contain all of these translations.

3. There are incorrect translations. For example, the one Kylie points out above, or the senior card action language, where they all say `applying` even when `renewing` or `replacing`.

We can resolve the first issue on our end by adding a script in the index.js file. Ultimately, how do we want to handle 2 and 3? Is the goal to have one small file for missing and incorrect translations that we send to translation services?

**Comment by baccigalupi** [comment]
The missing and incorrect translations can each go into a file in raw-data. The missing translations I was going to gather this week, but it will be ongoing. I don't expect you all to have to do that.

The real issue that we need to fix with this bug is the ways that the en.json file was changed over in the application. We need to fix those changes here. The goal is to get an en.json that matches the changes that we have done over in dlite-web, but these are programmatic with either new files or with scripts that change the json structure.

I put a lot of detail into the bug. Let me know if it is still unclear.

**Pull request closed without merging**